daemonset.yaml

```diff
@@ -63,7 +63,7 @@ spec:
         hostPath:
           path: /
       initContainers:
-      - image: gcr.io/cos-cloud/cos-gpu-installer:latest
+      - image: gcr.io/cos-cloud/cos-gpu-installer@sha256:8d86a652759f80595cafed7d3dcde3dc53f57f9bc1e33b27bc3cfa7afea8d483
```

Comment:
Could you add a comment of the driver installer version (v20200701)?

Reply:
Done